test/web-platform-tests/to-upstream/css/css-borders/border-style-length.html

```diff
@@ -36,4 +36,17 @@
   assert_equals(p.style.cssText, "border: 1px solid black;", "cssText");
   assert_equals(p.outerHTML, '<p style="border: 1px solid black;"></p>', "outerHTML");
 }, "Set border as 1px solid black");
+
+test(() => {
+  const div = document.createElement("div");
+  div.style.border = "1px solid #ffffff";
+  assert_equals(div.style.length > 0, true, "length");
+  assert_equals(div.style.border, "1px solid #ffffff", "border");
```

Comment:
This is incorrect. Browsers return `border: 1px solid rgb(255, 255, 255)`.

https://jsbin.com/zumasazure/edit?html,js,output